### DISCUSSION TITLE
ugs plataforma

### DISCUSSION THREAD

**Discussion by nando2608** [winder/Universal-G-Code-Sender#1731] [Help]
Hola, una consulta como solucionas este error que me aparece: (Múltiples códigos S en una línea.) En PLATAFORMA UGS, no carga los comandos de corte, espero me puedas ayudar gracias, saludos desde México.

**Comment by breiler** [comment]
Could you post your GCode file?
Which version of UGS are you running?

Your file either have a bad GCode format or you have encountered an old bug where UGS is trying to parse commands in comments.
